app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -0,0 +1,166 @@
+
+    return profitData
+
+  breakevenRevenueChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.product'),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.revenue'),"pattern":"","type":"number"}
+    ]
+
+    options = @defaultChartOptions
+    rows = []
+    for key,product of @breakevenRevenue
+      name = product.name
+      total = product.total
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
+
+    chartData = {"cols": columns, "rows": rows}
+    chartData = new google.visualization.DataTable(chartData);
+    chart = new google.visualization.PieChart(document.getElementById('breakeven-revenue-chart'));
+    chart.draw(chartData, options);
+
+  breakevenTotalCostsChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.item'),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.cost'),"pattern":"","type":"number"}
+    ]
+    rows = []
+
+    for key,product of @breakevenProductionCosts
+      name = product.name
+      total = product.total
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
+
+    for key,cost of @breakevenFixedCosts
+      name = cost.name
+      total = cost.amount
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
+
+    options = @defaultChartOptions
+    slices = {}
+
+    # Color is adjusted per item based on total items in a specific group
+    # Fixed costs have a different base color than product costs
+    productionCostLength = @breakevenProductionCosts.length
+    fixedCostLength = rows.length - productionCostLength
+    productionIncrement = parseInt((255-102)/productionCostLength)
+    fixedCostIncrement = parseInt((255-57)/fixedCostLength)
+
+    i = 0
+    for key,row of rows
+      if key < productionCostLength
+        slices[parseInt(key)] = {color:
+          "rgb(#{51 + (productionIncrement * parseInt(key))},
+          #{102 + (productionIncrement * parseInt(key))},
+          204)"
+        }
+      else
+        slices[parseInt(key)] = {color:
+          "rgb(220,
+          #{57 + (fixedCostIncrement * i)},
+          #{18 + (fixedCostIncrement * i)})"
+        }
+        ++i
+
+    options.slices = slices
+    chartData = {"cols": columns, "rows": rows}
+    chartData = new google.visualization.DataTable(chartData)
+    chart = new google.visualization.PieChart(document.getElementById('breakeven-total-costs-chart'))
+    chart.draw(chartData, options);
+
+  loadCharts: () ->
+    google.charts.load('current', {'packages':['corechart']});
+    google.charts.setOnLoadCallback @breakevenRevenueChart.bind @
+    google.charts.setOnLoadCallback @breakevenProductionCostsChart.bind @
+    google.charts.setOnLoadCallback @breakevenFixedCostsChart.bind @
+    google.charts.setOnLoadCallback @breakevenProductProfitChart.bind @
+    google.charts.setOnLoadCallback @breakevenTotalCostsChart.bind @
```

Comment:
Why the `bind @`'s here? Do you know about coffeescript's fat arrow `=>` ? But I would think neither should be necessary for top level functions like these.